.github/workflows/main.yml

```diff
@@ -131,11 +130,10 @@ jobs:
 #        run: codecov -f Snowflake.Data.Tests\${{ matrix.dotnet }}_${{ matrix.cloud_env }}_coverage.xml -t ffc6a21d-8176-4849-9047-e3a631dcd35a
```

Comment:
here I'm not asking to remove since I belive we will be adding it sooner or later